### DISCUSSION TITLE
🚀 (release): v1.7.3

### DISCUSSION THREAD

**Discussion by schlich** [dbt-msft/dbt-sqlserver#485] [General]
## What's Changed
* fix constraints support by @ms32035 in https://github.com/dbt-msft/dbt-sqlserver/pull/480
* Fix calc_batch_size maximum number of parameters allowed by @lucaslortiz in https://github.com/dbt-msft/dbt-sqlserver/pull/438
* v1.7.3 patch version bump by @schlich in https://github.com/dbt-msft/dbt-sqlserver/pull/483

## New Contributors
* @ms32035 made their first contribution in https://github.com/dbt-msft/dbt-sqlserver/pull/480
* @lucaslortiz made their first contribution in https://github.com/dbt-msft/dbt-sqlserver/pull/438

**Full Changelog**: https://github.com/dbt-msft/dbt-sqlserver/compare/v1.7.2...v1.7.3

<hr /><em>This discussion was created from the release <a href='https://github.com/dbt-msft/dbt-sqlserver/releases/tag/v1.7.3'>🚀 (release): v1.7.3</a>.</em>

**Comment by mmarqueee** [comment]
Hi there, thank you so much for the latest release. 

Is the is_inceremental() macro function fully implemented here? I have a problem with the incremental materializiations on the 1.7.2 

I checked up all the requirements to have the incremental materialization

- target table should exists
- materialized=incremental
- if is_incremental() is under a jinja syntax

but my if is_incremental() never compiles to a sql code. Here is the detailed description of the problem: https://github.com/dbt-msft/dbt-sqlserver/discussions/481

Thanks for you support
